### ISSUE TITLE
[CI] Multiple Failures in org.elasticsearch.xpack.ml.integration.PersistsJobIT

### ISSUE THREAD

**Issue by original-brownbear** [elastic/elasticsearch#41101]
I'm seeing the following on `master`:

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+multijob-unix-compatibility/os=oraclelinux-6/326/console

```
Execution failed for task ':x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner'.

    REPRODUCE WITH: ./gradlew :x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner --tests "org.elasticsearch.xpack.ml.integration.PersistJobIT.testPersistJobOnGracefulShutdown_givenTimeAdvancedAfterNoNewData" \
  -Dtests.seed=C09FCA0215302785 \
  -Dtests.security.manager=true \
  -Dtests.locale=en-BZ \
  -Dtests.timezone=America/Campo_Grande \
  -Dcompiler.java=12 \
  -Druntime.java=11

    REPRODUCE WITH: ./gradlew :x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner --tests "org.elasticsearch.xpack.ml.integration.PersistJobIT.testPersistJob" \
  -Dtests.seed=C09FCA0215302785 \
  -Dtests.security.manager=true \
  -Dtests.locale=en-BZ \
  -Dtests.timezone=America/Campo_Grande \
  -Dcompiler.java=12 \
  -Druntime.java=11

REPRODUCE WITH: ./gradlew :x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner --tests "org.elasticsearch.xpack.ml.integration.PersistJobIT.testPersistJobOnGracefulShutdown_givenTimeAdvancedAfterNoNewData" \
  -Dtests.seed=C09FCA0215302785 \
  -Dtests.security.manager=true \
  -Dtests.locale=en-BZ \
  -Dtests.timezone=America/Campo_Grande \
  -Dcompiler.java=12 \
  -Druntime.java=11

REPRODUCE WITH: ./gradlew :x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner --tests "org.elasticsearch.xpack.ml.integration.PersistJobIT.testPersistJob" \
  -Dtests.seed=C09FCA0215302785 \
  -Dtests.security.manager=true \
  -Dtests.locale=en-BZ \
  -Dtests.timezone=America/Campo_Grande \
  -Dcompiler.java=12 \
  -Druntime.java=11

REPRODUCE WITH: ./gradlew :x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner --tests "org.elasticsearch.xpack.ml.integration.PersistJobIT.testPersistJobOnGracefulShutdown_givenTimeAdvancedAfterNoNewData" \
  -Dtests.seed=C09FCA0215302785 \
  -Dtests.security.manager=true \
  -Dtests.locale=en-BZ \
  -Dtests.timezone=America/Campo_Grande \
  -Dcompiler.java=12 \
  -Druntime.java=11

REPRODUCE WITH: ./gradlew :x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner --tests "org.elasticsearch.xpack.ml.integration.PersistJobIT.testPersistJob" \
  -Dtests.seed=C09FCA0215302785 \
  -Dtests.security.manager=true \
  -Dtests.locale=en-BZ \
  -Dtests.timezone=America/Campo_Grande \
  -Dcompiler.java=12 \
  -Druntime.java=11

    REPRODUCE WITH: ./gradlew :x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner --tests "org.elasticsearch.xpack.ml.integration.ModelPlotsIT.testByFieldWithTerms" \
  -Dtests.seed=C09FCA0215302785 \
  -Dtests.security.manager=true \
  -Dtests.locale=nmg-CM \
  -Dtests.timezone=Asia/Kamchatka \
  -Dcompiler.java=12 \
  -Druntime.java=11

    REPRODUCE WITH: ./gradlew :x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner --tests "org.elasticsearch.xpack.ml.integration.ModelPlotsIT.testPartitionFieldWithoutTerms" \
  -Dtests.seed=C09FCA0215302785 \
  -Dtests.security.manager=true \
  -Dtests.locale=nmg-CM \
  -Dtests.timezone=Asia/Kamchatka \
  -Dcompiler.java=12 \
  -Druntime.java=11

REPRODUCE WITH: ./gradlew :x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner --tests "org.elasticsearch.xpack.ml.integration.ModelPlotsIT.testByFieldWithTerms" \
  -Dtests.seed=C09FCA0215302785 \
  -Dtests.security.manager=true \
  -Dtests.locale=nmg-CM \
  -Dtests.timezone=Asia/Kamchatka \
  -Dcompiler.java=12 \
  -Druntime.java=11

    REPRODUCE WITH: ./gradlew :x-pack:plugin:ml:qa:native-multi-node-tests:integTestRunner --tests "org.elasticsearch.xpack.ml.integration.ModelPlotsIT.testPartitionFieldWithTerms" \
  -Dtests.seed=C09FCA0215302785 \
  -Dtests.security.manager=true \
  -Dtests.locale=nmg-CM \
  -Dtests.timezone=Asia/Kamchatka \
  -Dcompiler.java=12 \
  -Druntime.java=11
```

Does not reproduce locally. Looks like there's a lot of logging about incrrect serialization in there:

```
[2019-04-11T03:10:28,702][WARN ][o.e.t.OutboundHandler    ] [node-0] send message failed [channel: Netty4TcpChannel{localAddress=/0:0:0:0:0:0:0:1:54038, remoteAddress=/0:0:0:0:0:0:0:1:44747}]
java.lang.IllegalStateException: Negative longs unsupported, use writeLong or writeZLong for negative numbers [-13000]
	at org.elasticsearch.common.io.stream.StreamOutput.writeVLong(StreamOutput.java:275) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.xpack.core.ml.job.process.autodetect.state.ModelSizeStats.writeTo(ModelSizeStats.java:154) ~[?:?]
	at org.elasticsearch.common.io.stream.StreamOutput.writeOptionalWriteable(StreamOutput.java:866) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.xpack.core.ml.action.GetJobsStatsAction$Response$JobStats.writeTo(GetJobsStatsAction.java:277) ~[?:?]
	at org.elasticsearch.common.io.stream.StreamOutput.lambda$writeCollection$28(StreamOutput.java:1073) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.common.io.stream.StreamOutput.writeCollection(StreamOutput.java:1092) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.common.io.stream.StreamOutput.writeCollection(StreamOutput.java:1073) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.common.io.stream.StreamOutput.writeList(StreamOutput.java:1080) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.xpack.core.action.util.QueryPage.writeTo(QueryPage.java:56) ~[?:?]
	at org.elasticsearch.xpack.core.ml.action.GetJobsStatsAction$Response.writeTo(GetJobsStatsAction.java:333) ~[?:?]
	at org.elasticsearch.transport.OutboundMessage.writeMessage(OutboundMessage.java:70) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.OutboundMessage.serialize(OutboundMessage.java:53) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.OutboundHandler$MessageSerializer.get(OutboundHandler.java:169) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.OutboundHandler$MessageSerializer.get(OutboundHandler.java:155) ~[elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.OutboundHandler$SendContext.get(OutboundHandler.java:202) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.OutboundHandler.internalSend(OutboundHandler.java:132) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.OutboundHandler.sendMessage(OutboundHandler.java:127) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.OutboundHandler.sendResponse(OutboundHandler.java:107) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.TcpTransportChannel.sendResponse(TcpTransportChannel.java:64) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.TaskTransportChannel.sendResponse(TaskTransportChannel.java:54) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.support.ChannelActionListener.onResponse(ChannelActionListener.java:47) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.support.ChannelActionListener.onResponse(ChannelActionListener.java:30) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.support.ContextPreservingActionListener.onResponse(ContextPreservingActionListener.java:43) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.xpack.ml.action.TransportGetJobsStatsAction.lambda$gatherStatsForClosedJobs$4(TransportGetJobsStatsAction.java:154) [x-pack-ml-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.xpack.ml.action.TransportGetJobsStatsAction.lambda$gatherDataCountsAndModelSizeStats$6(TransportGetJobsStatsAction.java:170) [x-pack-ml-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.xpack.ml.job.persistence.JobResultsProvider.lambda$modelSizeStats$26(JobResultsProvider.java:965) [x-pack-ml-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.xpack.ml.job.persistence.JobResultsProvider.lambda$searchSingleResult$28(JobResultsProvider.java:980) [x-pack-ml-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.ActionListener$1.onResponse(ActionListener.java:62) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.support.ContextPreservingActionListener.onResponse(ContextPreservingActionListener.java:43) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.support.TransportAction$1.onResponse(TransportAction.java:68) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.support.TransportAction$1.onResponse(TransportAction.java:64) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.support.ContextPreservingActionListener.onResponse(ContextPreservingActionListener.java:43) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.onResponse(AbstractSearchAsyncAction.java:316) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.onResponse(AbstractSearchAsyncAction.java:51) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.FetchSearchPhase$3.run(FetchSearchPhase.java:213) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.executePhase(AbstractSearchAsyncAction.java:166) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.executeNextPhase(AbstractSearchAsyncAction.java:159) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.ExpandSearchPhase.run(ExpandSearchPhase.java:120) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.executePhase(AbstractSearchAsyncAction.java:166) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.AbstractSearchAsyncAction.executeNextPhase(AbstractSearchAsyncAction.java:159) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.FetchSearchPhase.moveToNextPhase(FetchSearchPhase.java:206) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.FetchSearchPhase.lambda$innerRun$2(FetchSearchPhase.java:104) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.CountedCollector.countDown(CountedCollector.java:53) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.CountedCollector.onResult(CountedCollector.java:64) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.FetchSearchPhase$2.innerOnResponse(FetchSearchPhase.java:166) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.FetchSearchPhase$2.innerOnResponse(FetchSearchPhase.java:163) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.SearchActionListener.onResponse(SearchActionListener.java:45) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.SearchActionListener.onResponse(SearchActionListener.java:29) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.ActionListenerResponseHandler.handleResponse(ActionListenerResponseHandler.java:54) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.action.search.SearchTransportService$ConnectionCountingHandler.handleResponse(SearchTransportService.java:432) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleResponse(TransportService.java:1101) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.InboundHandler$1.doRun(InboundHandler.java:224) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.common.util.concurrent.EsExecutors$DirectExecutorService.execute(EsExecutors.java:192) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.InboundHandler.handleResponse(InboundHandler.java:216) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.InboundHandler.messageReceived(InboundHandler.java:141) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.InboundHandler.inboundMessage(InboundHandler.java:105) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.TcpTransport.inboundMessage(TcpTransport.java:644) [elasticsearch-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at org.elasticsearch.transport.netty4.Netty4MessageChannelHandler.channelRead(Netty4MessageChannelHandler.java:62) [transport-netty4-client-8.0.0-SNAPSHOT.jar:8.0.0-SNAPSHOT]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:323) [netty-codec-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:297) [netty-codec-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.handler.logging.LoggingHandler.channelRead(LoggingHandler.java:241) [netty-handler-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1436) [netty-handler-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1203) [netty-handler-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1247) [netty-handler-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:502) [netty-codec-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:441) [netty-codec-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:278) [netty-codec-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1434) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:965) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:163) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:656) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:556) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:510) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:470) [netty-transport-4.1.32.Final.jar:4.1.32.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:909) [netty-common-4.1.32.Final.jar:4.1.32.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

The log is over 200MB unfortunately so I can't attach it here.

**Comment by droberts195** [comment]
The underlying problem is the same as #41070, but in other tests.  From the server logs of this CI run:

```
bash-3.2$ find . -name '*.log' -exec grep 'Loading model snapshot' '{}' ';' | grep -v 'N/A'
[2019-04-11T00:41:18,200][INFO ][o.e.x.m.j.p.a.AutodetectProcessManager] [node-0] [test-start-datafeed] Loading model snapshot [-13] with latest_record_timestamp [2019-04-11T00:40:17.000Z], job latest_record_timestamp [2019-04-11T00:40:17.000Z]
[2019-04-11T02:58:55,103][INFO ][o.e.x.m.j.p.a.AutodetectProcessManager] [node-0] [time-advanced-after-no-new-data-test] Loading model snapshot [-13] with latest_record_timestamp [2019-04-11T03:43:53.000Z], job latest_record_timestamp [2019-04-11T03:43:53.229Z]
[2019-04-11T02:59:53,383][INFO ][o.e.x.m.j.p.a.AutodetectProcessManager] [node-0] [restore-model-snapshot-job] Loading model snapshot [-13] with latest_record_timestamp [2019-04-11T01:59:49.000Z], job latest_record_timestamp [2019-04-11T01:59:49.226Z]
[2019-04-11T03:03:52,213][INFO ][o.e.x.m.j.p.a.AutodetectProcessManager] [node-0] [detection-rules-it-test-condition] Loading model snapshot [-13] with latest_record_timestamp [2017-04-01T23:00:00.000Z], job latest_record_timestamp [2017-04-01T23:00:00.000Z]
[2019-04-11T03:07:10,887][INFO ][o.e.x.m.j.p.a.AutodetectProcessManager] [node-0] [interim-results-deleted-after-reopening-job-test] Loading model snapshot [-13] with latest_record_timestamp [2017-04-03T00:00:00.000Z], job latest_record_timestamp [2017-04-03T00:00:00.000Z]
[2019-04-11T02:46:53,330][INFO ][o.e.x.m.j.p.a.AutodetectProcessManager] [node-2] [mini-farequote-reopen] Loading model snapshot [-13] with latest_record_timestamp [2014-06-23T00:06:40.000Z], job latest_record_timestamp [2014-06-23T00:06:40.000Z]
```

Every model snapshot ID is `-13`.  There is something about the Oracle Linux 6 CI machines that causes this.  It started between builds 313 and 314, so between 19:38 and 23:19 UTC on 25th March.  I cannot see any code changes in ml-cpp, elasticsearch nor infra that explain this.  I think the only way forward is to get a debugging machine set up that has the same configuration as the Oracle Linux 6 CI machines.

**Comment by droberts195** [comment]
elastic/ml-cpp#459 should fix this.  I'll try to get that merged on master today and backported to 6.7 tomorrow morning, so please don't mute this test until after that as it only runs once every 12 hours and will validate the fix.  (If the tests continue to fail after tomorrow afternoon then we can assess how best to mute - given the nature of the problem it might be best not to run _any_ ML integration tests on Oracle Linux 6.)
